src/main/java/commands/HelpCommand.java

```diff
@@ -0,0 +1,17 @@
+package commands;
+
+import tasks.TaskList;
+import static messages.Message.MESSAGE_INSTRUCTIONS;
```

Comment:
Well done on abstracting out the MESSAGE_INSTRUCTIONS instead of putting them all here